src/main/java/org/betonquest/betonquest/menu/ItemDescription.java

```diff
@@ -55,7 +55,7 @@ public List<String> getLore(final Profile profile) {
         }
         final List<String> lore = new ArrayList<>(lines.size());
         for (final Line line : lines) {
-            lore.addAll(Arrays.asList(line.resolve(profile).split("\n")));
+            lore.addAll(Arrays.asList(line.resolve(profile).split("\n|\\\\n")));
```

Comment:
I think you found the right code snippet to fix the bugged behaviour from the variable objective. But instead we should fix the variable objective, as if you access it from PlaceholderAPI or an other place from BQ, it is still bugged